anomalib/models/cflow/config.yaml

```diff
@@ -4,11 +4,13 @@ dataset:
   path: ./datasets/MVTec
   category: bottle
   task: segmentation
-  image_size: 256
   train_batch_size: 16
   eval_batch_size: 16
   inference_batch_size: 16
   num_workers: 8
+  image_size: 256 # dimensions to which images are resized (mandatory)
+  center_crop: null # dimensions to which images are center-cropped after resizing (optional)
+  normalize: true # when true, images will be normalized to ImageNet statistics
```

Comment:
> `ImageNet statistics`

Using the torchfx extractor, the pre-trained model is using the new model versioning system in `torchvision`, no?

If that's the case, I think the preprocessing statistics necessary to each pre-trained model are encapsulated in the version object (?), so it shouldn't need to be assumed anywhere that it's from imagenet (which is often but not always the case).

Reply:
Another comment: I think consts/enums should be used instead of `bool`s; if a new option for normalization shows up, it's natural to add a value in the enum (while the bool would need a refactor,).

Do you think there could be a use case where the `normalize` needs more than 2 values? 
By contradiction, is it impossible that `normalize` needs more options than a `bool`?

Reply:
We default to ImageNet normalization because it's by far the most used. If a user needs custom normalization statistics they can configure it through the `transform_config`.

Reply:
> Another comment: I think consts/enums should be used instead of bools; if a new option for normalization shows up, it's natural to add a value in the enum (while the bool would need a refactor,).

Good idea. I'll replace the bool with an enum :)